.github/workflows/publish.yaml

```diff
@@ -0,0 +1,81 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+
+jobs:
+  # Run the CI tests
+  publish-stack:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Login to Docker Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ github.repository_owner }}
```

Comment:
Wait, who is the owner when it's an org? Can you test and find out please?

Reply:
If you use DOCKER_USERNAME it will just be easier

Reply:
secrets.DOCKER_USERNAME will cause every occurrence of the username to be starred out, which is annoying.

Reply:
We have that everywhere else and I don't think that openfaas can push images? I can't get a PTA for openfaas the owner.